tests/queries/0_stateless/02911_getHTTPHeaderFuncion.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+echo "SELECT getHTTPHeader('X-Clickhouse-User')" | curl -s -H 'X-ClickHouse-User: default' -H 'X-ClickHouse-Key: ' 'http://localhost:8123/' -d @-  
+
+echo "SELECT getHTTPHeader('X-Clickhouse-User'), getHTTPHeader('key1'), getHTTPHeader('key2')" | curl -s -H 'X-Clickhouse-User: default' \
+    -H 'X-ClickHouse-Key: ' -H 'key1: value1' -H 'key2: value2' 'http://localhost:8123/' -d @-
+
+echo "SELECT getHTTPHeader('X-Clickhouse-User'), getHTTPHeader('key1'), getHTTPHeader('key2')" | curl -s -H 'X-Clickhouse-User: default' \
+    -H 'X-ClickHouse-Key: ' -H 'key1: value1' -H 'key2: value2' 'http://localhost:8123/' -d @-
+
+echo "SELECT getHTTPHeader('X-' || 'Clickhouse' || '-User'), getHTTPHeader('key1'), getHTTPHeader('key2')" | curl -s -H 'X-Clickhouse-User: default' \
+    -H 'X-ClickHouse-Key: ' -H 'key1: value1' -H 'key2: value2' 'http://localhost:8123/' -d @-
+
+db_name=db_$(echo $RANDOM |md5sum |cut -c 1-4)
```

Comment:
no need to create and use database, for each test we already generate and use a unique database